introspection-engine/connectors/mongodb-introspection-connector/tests/dirty_data/mod.rs

```diff
@@ -103,7 +103,7 @@ fn mixing_types_with_the_same_base_type() {
     let expected = expect![[r#"
         model A {
           id    String @id @default(auto()) @map("_id") @db.ObjectId
-          /// Multiple data types found: Date: 50%, Timestamp: 50% out of 3 sampled entries
+          /// Multiple data types found: DateTime (Date): 50%, DateTime (Timestamp): 50% out of 3 sampled entries
```

Comment:
Do we even have to use `Json` here if both would map to `DateTime` in Prisma?

Reply:
Kind of... The parsing logic is different for date and timestamp...

Reply:
Right now it's safer to use Json...

Reply:
[`ba94f7f` (#2787)](https://github.com/prisma/prisma-engines/pull/2787/commits/ba94f7f30349a0b0bf2b8f69dca3288b236186ed)

Reply:
nvm